app/components/settings/billing-page/membership-section/status-pill.hbs

```diff
@@ -0,0 +1,20 @@
+<div
+  class="inline-flex items-center gap-2 rounded-full px-2 py-1 text-sm font-semibold
```

Comment:
Would move `text-sm` and `font-semibold` form here and keep them closer to the text (where we're currently applying `uppercase`). That way it's more clear what styles apply to the text and what styles apply to the pill container.